content/millennium/dstu2/conformance/structure-definition.md

```diff
@@ -53,7 +53,7 @@ Authorization is not required.
 
 ### Headers
 
-<%= headers head: {Accept: 'application/json+fhir'} %>
+<%= headers head: {'<a href="../../#media-types">Accept</a>': 'application/fhir+json'} %>
```

Comment:
for dstu2 must be json+fhir

Reply:
Made the change here: [Fix](https://github.com/cerner/fhir.cerner.com/pull/323/commits/2bcc448db093c8c5488198f21d9937fba354d60e#diff-eeb4d69334618c2654f5aee38ad9e21cL56)